### ISSUE TITLE
Unused Imports, code style, code cleanup

### ISSUE THREAD

**Issue by moshejs** [dancancro/great-big-example-application#194]
I've noticed there are a lot of unused imports, bad syntax (doesn't follow style guide, isn't consistent) across the entire project. I could sweep up some junk for this project. Let me know your thoughts

**Comment by dancancro** [comment]
Absolutely! This is a melting pot of approaches and libraries, so there's bound to be a lot of duplication and inconsistency. The end goal is to make it as DRY and uniform as possible. 

Since it's something every application has, one big area of contention is user data and services. Another one is configuration and application settings. I've been thinking about user/identity treatment a lot the past couple of days while working on incorporating the Thinkster blog app. It adds a bio field to the user object and relates user to other things. These are two things I haven't had to address yet in the context of JHipster. So I'm studying Spring JPA. Anyway, just keep that in mind as you find things to clean up. I'm not focused too much on configuration at the moment, so that might be a good area to tackle. Eliminating unused imports would be great too. Do you know an easy way to do that?

In general, I try to prioritize opinions of the Angular style guide first because it's the most official opinion on Angular and then JHipster because I want to be able to easily incorporate improvements as new versions come out. There's a couple of JHipster practices that aren't perfectly inline with the style guide. If you see any, I'd give those lower priority or add a proposed JHipster change to [this issue](https://github.com/jhipster/generator-jhipster/issues/6117).

**Comment by moshejs** [comment]
I could implement the Thinkster app, as have been using their code as reference for three projects in the past four months, I just need direction on where to place them

**Comment by dancancro** [comment]
Great. Well, I am pretty far along with it on the blog branch, but you're welcome to take over. Give me a couple days to get things in shape. The two big challenges are switching the front end to use ngrx and the whole backend, which doesn't have a Spring version yet.

**Comment by moshejs** [comment]
I don't believe I have the experience with nrgx, Spring, or JHispter. I reached out to you via email with more info.

**Comment by moshejs** [comment]
Hey Dan,
 I saw your questions on GitHub and I see what you're gauging with. I
understand the problems you're trying to handle with material and
bootstrap, and everyone has their own opinions about that.

I'm based in New York, so I can handle some high level conversations
tonight until 1am. I'm available tomorrow morning, and I should be lounging
all day, so any time really works for me.
